src/widgets/Array.vue

```diff
@@ -33,7 +33,7 @@
 					<b-tab
 						v-for="(child, index) in value"
 						style="{margin-top: 5px}"
-						:key="index"
+						:key="'tab-' + index"
```

Comment:
This might not be enough to make delete work correctly with multiple values

Reply:
reverted